docs: fix skaffold guide (#348)
hetznercloud/hcloud-cloud-controller-manager: README.md

kubectl -n kube-system create secret generic hcloud --from-literal="token=$HCLOUD_TOKEN"
```

7. Deploy the hcloud-cloud-controller-manager
```
SKAFFOLD_DEFAULT_REPO=your_docker_hub_username skaffold dev
```

- `docker login` required

> - Skaffold is using your own dockerhub repo to push the HCCM image.

Reviewer: ```suggestion
- Skaffold is using your own Docker Hub repo to push the HCCM image.
```

- After the first run, you might need to set the image to "public" on hub.docker.com

> On code change, skaffold will repack the image & deploy it to your test cluster again. Also, it is printing all logs from hccm components.

Reviewer: ```suggestion
On code change, Skaffold will repack the image & deploy it to your test cluster again. It will also stream logs from the hccm Deployment.
```
